setup.py

```diff
@@ -21,7 +21,7 @@
 from setuptools.command.sdist import sdist as sdist
 
 
-CYTHON_DEPENDENCY = 'Cython(>=0.29.24,<0.30.0)'
+CYTHON_DEPENDENCY = 'Cython(>=0.29.30,<0.30.0)'
```

Comment:
```suggestion
CYTHON_DEPENDENCY = 'Cython(>=0.29.32,<0.30.0)'
```
https://pypi.org/project/Cython